application/common/mail/ext-group-sync-errors.html.php

```diff
@@ -0,0 +1,44 @@
+<?php
+use yii\helpers\Html as yHtml;
+
+/**
+ * @var string $appPrefix
+ * @var string[] $errors
+ * @var string $googleSheetUrl
+ * @var string $emailSignature
+ * @var string $idpDisplayName
+ * @var string $idpName
+ */
+?>
+<p>
+    The following errors occurred when syncing the <?= yHtml::encode($appPrefix) ?>
+    external groups to the <?= yHtml::encode($idpDisplayName) ?> IDP:
+</p>
+<?= yHtml::ul($errors) ?>
+
+<?php
+if (empty($googleSheetUrl)) {
```

Comment:
Isn't it impossible for `googleSheetUrl` to be empty? It would at least have this much: 'https://docs.google.com/spreadsheets/d/'

Reply:
Ahh, good catch. I'll try to fix that.

Reply:
Submitted follow-up PR to fix that:
https://github.com/silinternational/idp-id-broker/pull/376